README.md

```diff
@@ -95,7 +164,7 @@ cmake --build ./build --target Gothic2Notr
 | `-w <worldname.zen>`   | startup world; newworld.zen is default                           |
 | `-save q`              | load the quick save on start                                     |
 | `-save <number>`       | load a specified save-game slot on start                         |
-| `-v -validation`       | enable Vulkan validation mode                                    |
+| `-v -validation`       | enable validation mode                                           |
```

Comment:
without mention of Vulkan it's not clear what sort of validation is it.
Maybe "enable validation layers for graphics api"